### PR TITLE
Add sv-parser

### PR DESCRIPTION
**Pull request by dalance**
I want to append sv-parser which is a SystemVerilog parser written in Rust.

https://github.com/dalance/sv-parser

### CODE REVIEW

**Comment by piotr-binkowski** [comment]
Hi @dalance
Your changes look all right but there is a small issue with your commit sign-off.
Could you modify it so that it contains your full name?
ie. `Signed-off-by: FIRST_NAME LAST_NAME <dalance@gmail.com>`

**Comment by mithro** [comment]
Hi,

Thank you for sending the pull request! It's awesome to see you contributing to this project. However, before I can look at this request you need to add [DCO sign-off](https://developercertificate.org/) to commits in this repository.

We require [DCO sign-off](https://developercertificate.org/) for **all** commits to this repository (including all authors). Please see the [following URL](https://github.com/probot/dco#how-it-works) to see more information and BeeWare also have a [Beginners guide to DCOs](https://pybee.org/contributing/how/dco/).

You can use `git commit --amend --signoff` and then a `git push --force` to add the signoff. You can also use `git rebase --signoff`.

Thanks!

**Comment by dalance** [comment]
Hi, @piotr-binkowski  @mithro 
Thanks. I fixed commit signoff.

File: tools/runners/sv_parser.py [hunk 1]
```diff
@@ -0,0 +1,15 @@
+from BaseRunner import BaseRunner
+
+
+class sv_parser(BaseRunner):
+    def __init__(self):
+        super().__init__("sv-parser", "parse_sv")
+
```
Comment (hzeller) [inline tools/runners/sv_parser.py]: If you add 
```
   self.url = "https://github.com/dalance/sv-parser"
```
.. the header link in the log-files will be pointing to your code.

Comment (dalance) [reply tools/runners/sv_parser.py]: Thanks. I'll add it.

**Pull request merged**